db/migrate/20150513082710_change_status_on_rents.rb

```diff
@@ -0,0 +1,20 @@
+class ChangeStatusOnRents < ActiveRecord::Migration
+  def up
+    change_column_default(:rents, :status, nil)
+    # Should this be done in raw SQL or smth instead?
+    # d.wessman 2015-05-13
+    Rent.find_each do |rent|
+      if rent.status == 'Ej bestämd' || rent.status == 'Ej bestÃƒÂ¤md'
```

Comment:
Ah. This is a simple problem to solve in SQL and there are many resources online. For this problem a ruby solution would also be acceptable but it's nicer to do it directly.

Send your diffs by me if you're worried about errors.